src/view/pie/PieChartLegend.svelte

```diff
@@ -0,0 +1,77 @@
+<script>
+	/**
+	 * This component creates a custom legend for the pie chart displaying all currently filtered status state along
+	 * with two links to set status state to all known entries and reset all filters.
+	 */
+	import { getContext } from "svelte";
+
+	import { localize } from "@typhonjs-fvtt/runtime/svelte/helper"
+
+	const spreadsheetStore = getContext("spreadsheetStore");
+	const { pieData, statuses } = spreadsheetStore.stores;
+</script>
+
+<section>
+	{#each $statuses as status, cntr (status.key)}
```

Comment:
What is `cntr` an abbreviation for?

Reply:
`counter`. I suppose you can say it's the idiomatic choice I have used my entire coding career. You can use `i` or `j` or the like or something else. Maybe `cntr` was more prevalent when I learned coding in the 90s not sure, but that is what I've used for loops and the like forever.